master/reference/calicoctl/resources/globalnetworkpolicy.md

```diff
@@ -6,13 +6,13 @@ no_canonical: true
 A global network policy resource (`GlobalNetworkPolicy`) represents an ordered set of rules which are applied
 to a collection of endpoints that match a [label selector](#selector).
 
-GlobalNetworkPolicy is not a namespaced resource. GlobalNetworkPolicy applies to [WorkloadEndpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint) in all namespaces, and to [HostEndpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint).
-Select a namespace in a GlobalNetworkPolicy in the standard selector by using
-`projectcalico.org/namespace` as the label name and a namespace name as the
+`GlobalNetworkPolicy` is not a namespaced resource. `GlobalNetworkPolicy` applies to [workload endpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint) in all namespaces, and to [host endpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint).
+Select a namespace in a `GlobalNetworkPolicy` in the standard selector by using
+`projectcalico.org/namespace` as the label name and a `namespace` name as the
 value to compare against, e.g., `projectcalico.org/namespace == "default"`.
-See [NetworkPolicy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy) for namespaced Network Policy.
+See [network policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy) for namespaced network policy.
```

Comment:
Hmm we're talking about `NetworkPolicy` the Calico resource here, not a generic `network policy` (word - which could mean any network policy k8s, calico or the generic word), so I prefer the original syntax. Thanks how other resource docs are also written. Maybe we should create a separate  PR to clean this up for all the resources (including backticks)?

Reply:
Actually, Kubernetes has a `NetworkPolicy`.

Reply:
I can do in a different PR I guess or else check now and see.